bbb-graphql-server/bbb_schema.sql

```diff
@@ -664,8 +664,12 @@ select
 	"user_voice"."talking",
     "user_voice"."startTime",
     "user_voice"."endTime",
-	"user_voice"."voiceActivityAt"
+	"user_voice"."voiceActivityAt",
+    "user"."color",
+    "user"."name",
+    "user"."speechLocale"
 FROM "user_voice"
+JOIN "user" ON "user"."userId" = "user_voice"."userId"
```

Comment:
I suggest to get User's infos using Hasura relationship instead of adding them to the view `v_user_voice_activity`.

Reply:
I tried doing that with Hasura relations but it was complaining `voiceActivityAt` field was not found in the `user` table. I think Hasura expects all tables in the relation to have the cursor fields.

Reply:
I tried doing that with Hasura relationships but it was complaining `voiceActivityAt` field was not found in the `user` table. I think Hasura expects all tables in the relation to have the cursor fields.

Reply:
You need to query `user` as child of `user_voice_activity_stream`!

```gql
subscription {
  user_voice_activity_stream(cursor: {initial_value: {voiceActivityAt: "2020-01-01"}}, batch_size: 10) {
    meetingId
    muted
    startTime
    endTime
    talking
    userId
    user {
      userId
      name
    }
  }
}
```

Just tested here and it works!